ocs_archive/settings/settings.py

```diff
@@ -44,6 +44,9 @@ def get_tuple_from_environment(variable_name, default):
 # Fits headers that must be present
 REQUIRED_HEADERS = get_tuple_from_environment('REQUIRED_HEADERS', 'PROPID,DATE-OBS,INSTRUME,SITEID,TELID,OBSTYPE,BLKUID')
 
+# Metadata that must be present when ingesting a thumbnail
+REQUIRED_THUMBNAIL_METADATA = get_tuple_from_environment('REQUIRED_THUMBNAIL_METADATA', 'frame_basename,size')
```

Comment:
I think we also need the site_id, instrument_id, and observation_day to be required so we know where to store/retrieve the thumbnails in the daydirs. Actually looking above I think we need to require observation_day for LCO fits files too since its required for that path!